app/components/post_list/post_list.js

```diff
@@ -58,18 +66,13 @@ export default class PostList extends PureComponent {
         if (item instanceof Date) {
             return item.getTime();
         }
-        if (item === General.START_OF_NEW_MESSAGES || item === LOAD_MORE_POSTS) {
-            return item;
-        }
 
-        return item.id;
+        return item.id || item;
     };
 
     loadMorePosts = () => {
```

Comment:
We can just call `this.props.loadMore` instead of having `this.loadMorePosts` call it